.github/dependabot.yml

```diff
@@ -1,6 +1,6 @@
 version: 2
 updates:
-- package-ecosystem: maven
+- package-ecosystem: github-actions
```

Comment:
This is incorrect in terms of CD

Reply:
Hi, @NotMyFault!

I just followed the instructions from here: https://www.jenkins.io/doc/developer/publishing/releasing-cd/

![image](https://github.com/jenkinsci/google-chat-notification-plugin/assets/8527036/08c08f35-95d2-4173-a901-0633bbf8cb9f)

Reply:
Oh, nevermind, I think it was my misinterpretation. I should add the github-actions configuration, not replace the original with it. I will fix it.

Reply:
I was about to say that but no worries :)